### PR TITLE
Sbt 1 setup spring mvc

### PR DESCRIPTION
**Pull request by luchikm86**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,34 +9,63 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+            <scope>test</scope>
```
Comment (bohdanvan) [inline pom.xml]: Why do we need this dependency? Remove, if unused.

File: pom.xml [hunk 2]
```diff
@@ -9,34 +9,63 @@
...
+        </dependency>
+    </dependencies>
+
+    <repositories>
+        <repository>
+            <id>spring-releases</id>
+            <url>https://repo.spring.io/libs-release</url>
+        </repository>
```
Comment (bohdanvan) [inline pom.xml]: Don't use other repositories, if you can use Maven Central. Delete, if this all dependencies are accessible from Maven Central.

File: pom.xml [hunk 3]
```diff
@@ -9,34 +9,63 @@
...
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>spring-releases</id>
+            <url>https://repo.spring.io/libs-release</url>
+        </pluginRepository>
```
Comment (bohdanvan) [inline pom.xml]: Don't use other repositories, if you can use Maven Central. Delete, if this all dependencies are accessible from Maven Central.

File: src/main/java/com/sobutylnik/hello/HelloWorld.java [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
 public class HelloWorld {
     public static void main(String[] args) {
-        Runnable helloPrinter = () -> System.out.println("Hello");
-        helloPrinter.run();
+
+//        Runnable helloPrinter = () -> System.out.println("Hello");
+//        helloPrinter.run();
```
Comment (bohdanvan) [inline src/main/java/com/sobutylnik/hello/HelloWorld.java]: Drop this code. It was used only for Java 8 testing.

File: src/main/java/com/sobutylnik/hello/HelloWorld.java [hunk 5]
```diff
@@ -1,8 +1,15 @@
 package com.sobutylnik.hello;
 
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
 public class HelloWorld {
```
Comment (bohdanvan) [inline src/main/java/com/sobutylnik/hello/HelloWorld.java]: Rename to Main.

File: src/main/java/com/sobutylnik/hello/IndexController.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class IndexController {
+    private static final String hello = "Hello";
+
+    @RequestMapping("/")
+//    @RequestMapping(value = "/index", produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (bohdanvan) [inline src/main/java/com/sobutylnik/hello/IndexController.java]: Remove unused code.

File: src/main/java/com/sobutylnik/hello/IndexController.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+package com.sobutylnik.hello;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class IndexController {
+    private static final String hello = "Hello";
```
Comment (bohdanvan) [inline src/main/java/com/sobutylnik/hello/IndexController.java]: No need for this constant.
PS. Constants in Java are declared in UPPER_CASE.

File: pom.xml (hunk 1 above)
Comment (luchikm86) [reply pom.xml]: +

**Pull request merged**